cql3/expr/expression.hh

```diff
@@ -379,6 +480,27 @@ extern expression replace_column_def(const expression&, const column_definition*
 // For example this changes token(p1, p2) < token(1, 2) to my_column_name < token(1, 2).
 extern expression replace_token(const expression&, const column_definition*);
 
+
+extern ::shared_ptr<term> prepare_term(const expression& expr, database& db, const sstring& keyspace, const column_specification_or_tuple& receiver);
```

Comment:
`extern` is not needed in `expression.hh`

Reply:
Last time I didn't add it @dekimir complained, so I added it.